provision/acc_provision/acc_provision.py

```diff
@@ -538,9 +541,33 @@ def config_adjust(args, config, prov_apic, no_random):
         adj_config["aci_config"]["vmm_domain"]["injected_cluster_type"] = ""
     if not config["aci_config"]["vmm_domain"].get("injected_cluster_provider"):
         adj_config["aci_config"]["vmm_domain"]["injected_cluster_provider"] = ""
+
+    if config["sriov_config"].get("enable") is True:
+        adj_config["vendors"] = ""
+        adj_config["devices"] = ""
+        device_info = get_device_info()
+        if device_info is not None:
+            adj_config["vendors"] = device_info[0]
+            adj_config["devices"] = device_info[1]
+
     return adj_config
 
 
+def get_device_info():
```

Comment:
I am not sure I understand this approach. This function is obtaining information from the host on which its running (you are probably making the assumption that it's a node in the cluster). However, acc-provision can be run from any place, and in fact, the cluster node won't even be ready when acc-provision is run. The job of acc-provision is to program the APIC and generate the ACI CNI deployment spec prior to the cluster install.

Reply:
That makes sense. If we are not able to populate  vendor and device info of the Nic Cards as part of acc_provision run, we have to manually put in those information to  sriov network plugin configmap. Madhu and I will catch up with you to think about a better approach.